.github/workflows/bicep_deploy.yml

```diff
@@ -127,9 +127,18 @@ jobs:
       resourceGroupName: ${{ inputs.resourceGroupName }}
       action: validate
     secrets: inherit
+    
+  what_if:
+    needs: validate
+    uses: ./.github/workflows/bicep_az_deployment_group.yml
+    with:
+      environment: ${{ inputs.environment }}
+      resourceGroupName: ${{ inputs.resourceGroupName }}
+      action: what-if
+    secrets: inherit
 
   deploy:
-    needs: validate
+    needs: what_if
```

Comment:
was there a way for this to be manual only for certain environments and not for others?

Reply:
I've left it as automatic for all environments for now as it looked quite complex to add the manual step so will come back and look at it if I get more time

Reply:
Sounds good, will resolve :)